runtime/testy/studentCode.py

```diff
@@ -166,11 +175,36 @@ def emergencyStop_main():
   Robot.setValue(response, "incrementer")
   print("HIBIKE LOOP")
 
-def hibikeSensorMappings_setup():
-  pass
-
-def hibikeSensorMappings_main():
-  print(Robot._hibikeGetUID('zero'))
-  print(Robot._hibikeGetUID('one'))
-  print(Robot._hibikeGetUID('two'))
-  print(Robot._hibikeGetUID('three'))
+# Due to interaction with hibike virtual devices, currently
```

Comment:
Would it be possible for you to modify the `test` target in the runtime Makefile to programmatically execute these tests? (You can add calls to `spawn_virtual_devices.py` in the makefile)